README.md

```diff
@@ -150,6 +153,28 @@ To use the bot without deploying your own instance, follow these steps:
     <https://coqbot.herokuapp.com/job> that will only be triggered by
     job events.
```

Comment:
This step can now be simplified: the user can create a single webhook (`/gitlab`) triggering both pipeline and job events.